interaction.js

```diff
@@ -462,3 +463,31 @@ for (let i = 0; i < xicon.length; i++) {
     document.querySelector('.popscont').style.display = 'none';
   });
 }
+
+// ::::::::::::::::::::::Validation Form::::::::::::::::::::::::::::
+
+const $name = document.getElementById('name');
+const $email = document.getElementById('email');
+const $message = document.getElementById('message');
+$name.required = true;
+$email.required = true;
+$message.required = true;
+$email.pattern = '^(([^<>()\[\]\\.,;:\s@\"]+(\.[^<>()\[\]\\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()\[\]\\.,;:\s@\"]+\.)+[^<>()\[\]\\.,;:\s@\"]{2,})$';
+function validation() {
+  if ($email.value === $email.value.toLowerCase()) {
+    return true;
+  }
+  return false;
+}
+
+document.addEventListener('submit', (e) => {
+  if (e.target.matches('form')) {
+    if (validation()) {
+      console.log('Form submitted successfully!');
+    } else {
+      e.preventDefault();
+      alert('Please enter your email in lower case');
+      $email.value = $email.value.toLowerCase();
```

Comment:
- [ ] From the lessons previously given to you, it's required that we follow [JavaScript Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) and avoid using alert() and others you will find in that link. Kindly display an error text message close to the button when there is an uppercase letter in the email just as the requirement instructed and not alert(). 👍🏽

Reply:
Done @AmaduKamara! Thanks for your feedback. I've already made the changes.